src/components/Dropdown/V2/Dropdown.utils.ts

```diff
@@ -336,3 +359,10 @@ export const filterNonStoreProps = props => {
     return nextProps
   }, {})
 }
+
+export const isSelectedItemEmpty = selectedItem => {
+  if (selectedItem == null) return true
```

Comment:
@tinkertrain We have some handy `is` based helpers from `utilities/is`.. stuff like `isDefined`, `isArray`. It might help here 💪

Reply:
Handy!